osu.Desktop.VisualTests/Tests/TestCaseDetails.cs

```diff
@@ -0,0 +1,60 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Primitives;
+using osu.Framework.Testing;
+using osu.Game.Database;
+using osu.Game.Screens.Select;
+using System;
+using System.Linq;
+
+namespace osu.Desktop.VisualTests.Tests
+{
+    internal class TestCaseDetails : TestCase
+    {
+        private Details details;
+
+        public override void Reset()
+        {
+            base.Reset();
+
+            Add(details = new Details

```

Comment:
Would rename Details to BeatmapDetails or BeatmapDetailsPanel. Details is way too generic (and rename the test case to fit this).